apps/nextjs/src/components/utils.tsx

```diff
@@ -0,0 +1,33 @@
+import type { AlertProps } from "@mantine/core";
```

Comment:
I'm not so much a fan of this:
![image](https://github.com/homarr-labs/homarr/assets/63781622/e9b3fff6-2912-4c56-8097-b659664f1a70)

I like a simple notification more, do we need such an error display?

Reply:
I can reformat it to an error at put it in the lib files close to the `showSuccessNotification` . I also think a notification makes more sense but I'm afraid some people might miss it. I will try some experiments to make it really visible